modules/exploits/multi/http/apache_jetspeed_upload.rb

```diff
@@ -0,0 +1,195 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ManualRanking
```

Comment:
@wvu-r7 ManualRanking means your exploit is either unreliable or something has to be done manually. Like for example: it fails to give you a session most of the time, or it needs to user to click on a prompt to permit the exploit to run. The description only mentions the exploit can be destructive. Is this the reason why you wanted to lower it?